Formula/cockroach.rb

```diff
@@ -0,0 +1,82 @@
+class Cockroach < Formula
+  desc "Distributed SQL database"
+  homepage "https://www.cockroachlabs.com"
+  url "https://binaries.cockroachdb.com/cockroach-v1.0.src.tgz"
+  version "1.0"
+  sha256 "ca87b10eec688195e0df4f85431b019f2980ae4b511ee321f91f945315efeb76"
+  head "https://github.com/cockroachdb/cockroach.git"
+
+  depends_on "cmake" => :build
+  depends_on "go" => :build
+  depends_on "xz" => :build
+
+  def install
+    system "make", "install", "prefix=#{prefix}"
+  end
+
+  def caveats; <<-EOS.undent
+    CockroachDB is a distributed database intended for multi-server deployments.
```

Comment:
This first line can probably be removed (we try to keep caveats as small as possible since the longer they are, the less likely people are to read any of them).